config/make-ramcloud-container-test

```diff
@@ -0,0 +1,5 @@
+#/bin/bash -ex
```

Comment:
This line should have an exclamation point, i.e. it should be `#!/bin/bash -ex` instead of simply `#/bin/bash -ex`. Note, this may be true of other scripts in the same directory. It's probably worth looking at each script to see if the [shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) is well-formed.

Reply:
It's just the two test scripts that suffer this issue. Corrected both.